source/about/cloud-subscriptions.rst

```diff
@@ -177,4 +176,6 @@ These offerings are deployed in the following ways:
 Can I cancel and delete my Cloud workspace?
```

Comment:
@johndavidlugtu, was there a decision around whether the customer portal account deletion is offered? And what messaging do end-users see when they try to access the deleted workspace?

Reply:
Yep, the decision was to not offer it at this point, as we've theorized that the only a very few subset of workspace deletion requests genuinely want their portal account deletion. Since it was offered as a question from support, we suspected users just marked it as "Yes"